source/fundamentals/collations.txt

```diff
@@ -0,0 +1,539 @@
+==========
+Collations
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+This guide shows you how to use **collations** with MongoDB to order your
+query or aggregate results by string values. A collation is a set of
```

Comment:
```suggestion
query or aggregation results by string values. A collation is a set of
```

Aggregate vs Aggregation terminology. If you think aggregate is more appropriate here totally understand, but seems related to https://github.com/mongodb/docs-java/pull/70#discussion_r605236948

Reply:
Good catch, I'll update.